javascript/ola/LICENCE

```diff
@@ -1,13 +0,0 @@
-This program is free software; you can redistribute it and/or modify
```

Comment:
Can you resurrect this please. At least olad/www/js/app.js and any other code ought to have a license at the top which the presence of this file checks for.